### ISSUE TITLE
Remove placeholder illustrations

### ISSUE THREAD

**Issue by jkwnki** [jacobwhall/marktext#9]
Hi,

can you please remove the annoying pictures in the app, or at least add an option to hide those. Placing the green buttons ("New File" and so on) somewhere at top right or bottom right would be okay and would keep it clean and simple.
was impressed by the simplicity of this app until i came across that unnecessary beauty.

Thank you !!

![20230706_0204_screenshot_sel](https://github.com/jacobwhall/marktext/assets/58271617/5e3b225e-7544-4c0a-9091-6b8b954dc228)

![20230706_0202_screenshot_sel](https://github.com/jacobwhall/marktext/assets/58271617/3ad5d1e4-45c6-4790-bfb0-273119ef1512)

**Comment by jacobwhall** [comment]
Hey @jkwnki, thanks for bringing this up. To be honest with you these pictures don't bother me much, but I can't help but agree with your "unnecessary beauty" assessment. This wouldn't be a particularly difficult change, so if anyone's looking to get involved, I'm happy to review any pull requests.

**Comment by jacobwhall** [comment]
The first image you reference appears when all tabs have been closed in MarkText. It may be worth discussing what else should appear there. Perhaps the last tab shouldn't be closable (without closing the entire window), or perhaps we should display something else on the otherwise blank screen.

**Comment by jkwnki** [comment]
Hi,

> Perhaps the last tab shouldn't be closable

is a good idea. Maybe we could start like this:

![11](https://github.com/jacobwhall/marktext/assets/58271617/67710874-f056-45aa-b2b4-87692131b32a)

As we already have the filename in the tab, we could also change the window title to the appname (marktext) or the fullpath.

The second picture in my first post appears when a folder is opened to the side pane and the search or table of contents (H) are activated. Lets get rid of the unnecessary beauty! :)

When the app initially is started and a file is opened, marktext doesnt show a tab. It brings up tabs after a second file is opened.
Maybe we should consider the bring up the tab view when the first file is opened. 

Additionally when resizing the window in width, the content could be stretched more to left and right due to unnecessary margin.

The app also asks for file save even when file is unchanged.

A new app icon would be nice too :)

Greetings
jkwnki

**Comment by jacobwhall** [comment]
>> Perhaps the last tab shouldn't be closable

I've decided that this should be another issue, feel free to open one if you like.

> As we already have the filename in the tab, we could also change the window title to the appname (marktext) or the fullpath.

>  Maybe we should consider the bring up the tab view when the first file is opened.

Hmm, I might be ok with this change, but I'd want to discuss this a bit further (again, in a separate issue). It's nice when you first open MarkText, and there are no tabs in sight. Speaking of "unnecessary beauty" 😉

> Additionally when resizing the window in width, the content could be stretched more to left and right due to unnecessary margin.

> The app also asks for file save even when file is unchanged.

> A new app icon would be nice too :)

Sorry to make you jump through hoops to get your concerns addressed, but I'd prefer we open separate issues for these, too. Helps me keep track of what still needs to be done, and to have more focused conversations on a per-issue basis. At first glance, my responses are **a)** I like the margins but maybe we could make this user-customizable **b)** that is a bug that deserves fixing! **c)** I'm open to this, but let's get some more community engagement / buy-in before a wholesale rebranding of the project

I went ahead and removed the placeholder images in my latest commit. I'll post a new release tonight with builds for Linux. Let me know what you think! I added a little message to the page when you close every tab...I'm closing this issue but happy to revisit if it doesn't feel quite right
